zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -546,12 +546,26 @@ where
                     .memo
                     .as_ref()
                     .map_or_else(MemoBytes::empty, |m| m.clone());
-                builder.add_sapling_output(
-                    external_ovk,
-                    *ua.sapling().expect("TODO: Add Orchard support to builder"),
-                    payment.amount,
-                    memo.clone(),
-                )?;
+
+                if ua.sapling().is_some() {
+                    builder.add_sapling_output(
+                        external_ovk,
+                        *ua.sapling().unwrap(),
```

Comment:
```suggestion
                        sapling_reciever,
```
(may require a `*`, can't tell from here)